src/Http/Url.php

```diff
@@ -68,9 +68,8 @@ public function match($patterns)
             $patterns = [$patterns];
         }
 
-        //Remove scheme and query
-        $url = preg_replace('|(\?.*)?$|', '', (string) $this);
-        $url = preg_replace('|^(\w+://)|', '', $url);
+        //Remove scheme
+        $url = preg_replace('|^(\w+://)|', '', (string) $this);
```

Comment:
I've changed `match()` to support query parameteres, but it must be specified explicity using the `?` character, as explained here: https://github.com/oscarotero/Embed/pull/221#pullrequestreview-44115897

Can you update the pull request with this change and then update the black list patterns to include the `?`?